iaslic-bulletin.csl

```diff
@@ -0,0 +1,397 @@
+    <link href="http://www.zotero.org/styles/elsevier-with-titles" rel="template"/>
+    <link href="https://www.iaslic1955.org.in/fckeditor/userfiles/file/Publication_policy_Authorship_Manual_final.pdf" rel="documentation"/>
+    <author>
+      <name>Dr. Aditya Tripathi</name>
+      <email>aditya@bhu.ac.in</email>
+    </author>
+    <author>
+      <name>Nikhil Anand Srivastava</name>
+      <email>nikhil@bhu.ac.in</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <category field="humanities"/>
+    <issn>0018-8441</issn>
+    <summary>IASLIC citation style for Library and Information Science research articles, version 1.</summary>
+    <updated>2025-07-31T07:47:03+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="retrieved">Retrieved on</term>
+      <term name="volume" form="short">V</term>
+      <term name="editor" form="short">
+        <single>Ed.</single>
+        <multiple>Ed.</multiple>
+      </term>
+      <term name="page" form="short">p</term>
+      <term name="number-of-pages" form="short">p</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name delimiter=", " and="text" name-as-sort-order="all">
+            <name-part name="family" text-case="uppercase"/>
+            <name-part name="given" prefix=" (" suffix=")"/>
+          </name>
+          <label form="short" prefix=" (" suffix=")"/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name delimiter=", " and="text" name-as-sort-order="all">
+            <name-part name="family" text-case="uppercase"/>
+            <name-part name="given" prefix=" (" suffix=")"/>
+          </name>
+        </names>
+        <text term="editor" form="short" prefix=", " font-style="italic"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name form="long" name-as-sort-order="all" delimiter=" and ">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given" prefix=" (" suffix=")"/>
+      </name>
+      <label form="short" prefix=", " text-case="capitalize-first"/>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book article-newspaper entry-dictionary entry-encyclopedia" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else-if type="chapter paper-conference" match="any">
+        <text variable="title"/>
+      </else-if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" font-style="italic"/>
```

Comment:
I try to avoid macros that just call a variable. Please remove and just call the variable where you need it.